demonstrations/tutorial_falqon.py

```diff
@@ -0,0 +1,452 @@
+r"""
+Feedback-Based Quantum Optimization (FALQON)
+=========================
+
+.. meta::
+    :property="og:description": Solve combinatorial problems with no classical optimizer
+    :property="og:image": https://pennylane.ai/qml/_images/qaoa_layer.png
+
+.. related::
+
+   tutorial_qaoa_intro Intro to QAOA
+   tutorial_qaoa_maxcut QAOA for MaxCut
+
+*Authors: David Wakeham and Jack Ceroni. Posted: XXX. Last updated: XXX.*
+
+-----------------------------
+
+While the `Quantum Approximate Optimization Algorithm (QAOA) <https://pennylane.ai/qml/demos/tutorial_qaoa_intro.html>`__
+is one of the best-known processes for solving combinatorial optimization problems with quantum computers,
+it has drawbacks. Since QAOA relies on optimization over a parameter space,
+convergence isn't guaranteed, and the optimization procedure can become "stuck" in local minima.
+
+.. figure:: ../demonstrations/falqon/global_min.png
+    :align: center
+    :width: 70%
+
+In this demo, we'll be implementing the FALQON algorithm: a feedback-based algorithm
+for quantum optimization, introduced by `Magann, Rudinger, Grace & Sarovar (2021) <https://arxiv.org/pdf/2103.08619.pdf>`__.
+It is similar in spirit to QAOA, but instead uses iterative feedback steps rather than a global optimization
+over parameters, therefore avoiding the issue of local minima. We will show how to implement FALQON in PennyLane
+and test its performance on the MaxClique problem in graph theory!
+
+Theory
+------
+
+To solve combinatorial optimization problems using a quantum computer, a typical strategy is to encode
+the solution to the problem as the ground state of *cost Hamiltonian* :math:`H_c`, and then use some procedure to drive
+the system from an initial state into this ground state. FALQON falls under this broad scheme.
+
+Imagine a quantum system governed by a Hamiltonian of the form :math:`H = H_c + \beta(t) H_d`. These kinds of
+Hamiltonians come up quite often in the theory of `quantum control <https://quantiki.org/wiki/quantum-control-theory>`__,
+which studies how we may go about driving a quantum system from one state to another. The choice of :math:`\beta(t)` allows
+us to decide which state we want a system governed by such a Hamiltonian to evolve towards.
+
+The time-dependent Schrodinger equation tells us that the dynamics of the system are given by:
+
+.. math:: i \frac{d}{dt} |\psi(t)\rangle = (H_c + \beta(t) H_d) |\psi(t)\rangle
+
+where we set :math:`\hbar = 1`. Now, let us suppose that the objective of our quantum control experiment is to drive our system
+to the state :math:`|\psi\rangle`: the ground state of :math:`H_c`. Phrased differently, we would like to minimize the expectation
+value :math:`\langle H_c\rangle`. Therefore, a reasonable goal is to construct our system such that the expectation decreases with time:
+
+.. math:: \frac{d}{dt} \langle H_c\rangle_t = \frac{d}{dt} \langle \psi(t)|H_c|\psi(t)\rangle = i \beta(t)\langle [H_d, H_c] \rangle_t \leq 0
```

Comment:
```suggestion
.. math:: \frac{d}{dt} \langle H_c\rangle_t = \frac{d}{dt} \langle \psi(t)|H_c|\psi(t)\rangle = i \beta(t)\langle [H_d, H_c] \rangle_t \leq 0,
```
and similarly elsewhere make sure to add punctuation to equations too